.travis.yml

```diff
@@ -498,6 +498,7 @@ script:
 
     # Build and execute all examples.
     - ./travis/compileExec.sh "example/vectorAdd" ./vectorAdd
+    - ./travis/compileExec.sh "example/helloWorld" ./helloWorld
```

Comment:
I would sort them alphabetically or by difficulty. In either case yours would be the first!